src/renderers/webgl/WebGLTextures.js

```diff
@@ -650,15 +650,29 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 			glInternalFormat = _gl.DEPTH_COMPONENT;
 
-			if ( texture.type === FloatType ) {
+			if ( isWebGL2 ) {
+
+				if ( texture.type === FloatType ) {
+
+					glInternalFormat = _gl.DEPTH_COMPONENT32F;
+
+				} else if ( texture.type === UnsignedIntType ) {
 
-				if ( isWebGL2 === false ) throw new Error( 'Float Depth Texture only supported in WebGL2.0' );
-				glInternalFormat = _gl.DEPTH_COMPONENT32F;
+					glInternalFormat = _gl.DEPTH_COMPONENT24;
 
-			} else if ( isWebGL2 ) {
+				} else {
+
+					glInternalFormat = _gl.DEPTH_COMPONENT16; // WebGL2 requires signed internalformat for glTexImage2D
```

Comment:
Potential typo: shouldn't it be "sized" instead of "signed"?

Reply:
Indeed!